src/server/server_family.cc

```diff
@@ -502,6 +506,25 @@ void ServerFamily::Init(util::AcceptServer* acceptor, std::vector<facade::Listen
     return res.has_value();
   });
 
+  // We only reconfigure TLS when the 'tls' config key changes. Therefore to
+  // update TLS certs, first update tls_cert_file, then set 'tls true'.
```

Comment:
is it enough to issue CONFIG SET tls true to reload certificates?

Reply:
yep